src/client/pydaos/pydaos_shim.c

```diff
@@ -98,9 +101,29 @@ static PyObject *
 __shim_handle__daos_init(PyObject *self, PyObject *args)
 {
 	int rc;
+	int ret;
+	char *provider;
+	char *override;
 
 	rc = daos_init();
 
+	/* Event Queue creation is expensive with verbs provider */
+	if ((rc == 0) && (use_glob_eq == 0)) {
+		provider = getenv("CRT_PHY_ADDR_STR");
```

Comment:
I would honestly not check for the provider and always enable the global EQ approach by default. I would keep the override though for testing purpose.

Reply:
yes agreed that 1 global EQ should be the default regardless of provider. otherwise PR looks good to me